orangecontrib/spectroscopy/widgets/owpeakfit.py

```diff
@@ -403,6 +402,20 @@ def on_exception(self, ex):
             self.Outputs.residuals.send(None)
             self.Outputs.annotated_data.send(None)
 
+    @classmethod
+    def migrate_preprocessor(cls, preprocessor, version):
+        name, settings = preprocessor
+        settings = settings.copy()
+        if version < 2:
+            for n, h in settings.items():
+                if isinstance(h, dict):
+                    h = h.copy()
+                    set_default_vary(h)
+                    #FIX for vary fixed!!!!!
```

Comment:
Is this fix still needed?

Reply:
Thanks, I wrote this down as a note but then found out that the above function already handles it.